cmd/zed/dev/indexfile/section/command.go

```diff
@@ -23,7 +23,7 @@ writes it to the output.  The -trailer option writes
 the Zed index trailer to the output in addition to the section if the section
 number was specified.
 
-See the "zed index" command help for a description of a Zed index file.`,
+See the "zed dev indexfile" command help for a description of a Zed index file.`,
```

Comment:
I can't leave a comment at the site, but there's an error string down below that starts with "zed index section". I'd remove it, but if you keep it, change index to indexfile.